views/Feed/Feed.jsx

```diff
@@ -140,18 +140,14 @@ export default class FeedView extends React.Component {
   }
 
   updatePost = async (postId, data, type) => {
-
-    const {
-      navigation,
-    } = this.props;
-
     if (type === 'toggleLike') {
-      this.setState({
-        posts: this.state.posts.map(post => {
-          if (post._id === postId) return data;
-          return post;
+      return ApiClient.post(`/posts/${postId}/like`, {'addLike': data.isLiked}, {authorized: true})
+        .then(() => {
+          this.updateState('updatePost', postId);
         })
-      });
+        .catch(err => {
+          alert("Error deleting post. Sorry about that!")
```

Comment:
And maybe change this to 'liking'/'unliking post' based on `data.isLiked`

Reply:
Oops, reveals where I copy and pasted my code from :P

Reply:
;)